containers/layouts/TextDirectionSelect.js

```diff
@@ -2,21 +2,21 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { c } from 'ttag';
 import { Select } from 'react-components';
+import { RIGHT_TO_LEFT } from 'proton-shared/lib/constants';
+
+const { ON, OFF } = RIGHT_TO_LEFT;
 
 const TextDirectionSelect = ({ rightToLeft, onChange, loading }) => {
     const options = [
-        { text: c('Option').t`Left to Right`, value: 0 },
-        { text: c('Option').t`Right to Left`, value: 1 }
+        { text: c('Option').t`Left to Right`, value: OFF },
```

Comment:
Rename constants `LEFT_TO_RIGHT` and `RIGHT_TO_LEFT`.

Reply:
ok!